include/powsybl/iidm/NetworkIndex.hxx

```diff
@@ -39,6 +39,9 @@ T& NetworkIndex::checkAndAdd(std::unique_ptr<T>&& identifiable) {
     }
 
     auto ptrIdentifiable = std::unique_ptr<Identifiable, Deleter>(identifiable.release());
+    for (const std::string& alias : ptrIdentifiable->getAliases()) {
```

Comment:
Notice : this code is not really needed because aliases are added after instantiation and manually by calling `Identifiable::addAlias()`